src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagerReconfigureTaskProviderTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.provider;
+
+import au.com.dius.pact.provider.junit5.PactVerificationContext;
+import au.com.dius.pact.provider.junit5.PactVerificationInvocationContextProvider;
+import au.com.dius.pact.provider.junitsupport.Provider;
+import au.com.dius.pact.provider.junitsupport.State;
+import au.com.dius.pact.provider.spring.junit5.MockMvcTestTarget;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootContractProviderBaseTest;
+import uk.gov.hmcts.reform.wataskmanagementapi.controllers.TaskActionsController;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+@Provider("wa_task_management_api_reconfigure_task_by_case_id")
+public class TaskManagerReconfigureTaskProviderTest extends SpringBootContractProviderBaseTest {
+
+    @TestTemplate
+    @ExtendWith(PactVerificationInvocationContextProvider.class)
+    void pactVerificationTestTemplate(PactVerificationContext context) {
+        if (context != null) {
+            context.verifyInteraction();
+        }
+    }
+
+    @BeforeEach
+    void beforeCreate(PactVerificationContext context) {
+        MockMvcTestTarget testTarget = new MockMvcTestTarget();
+        testTarget.setControllers(new TaskActionsController(
+            taskManagementService,
+            accessControlService,
+            systemDateProvider,
+            clientAccessControlService
+        ));
+        if (context != null) {
+            context.setTarget(testTarget);
+        }
+
+    }
+
+    @State({"reconfigure tasks using caseId"})
```

Comment:
This will fail pact verification as state does not match consumer reference. Should be "reconfigure a task using caseId".